pysat/__init__.py

```diff
@@ -84,6 +84,6 @@
 from ._custom import Custom
 from ._orbits import Orbits
 from . import instruments
-from . import ssnl
+import pysatSeasons as ssnl
```

Comment:
What if we put this in a try: except: now?  

If a user has pysatSeasons installed and working then it is available in pysat.

Reply:
I think we need to discuss at the next meeting.  What I would like to see for 2.1.0 is deprecation warnings on every function that will be dropped in 3.0.  The import of pysatSeasons (soft or otherwise) might be dropped completely, and just warn users that this will change.  If we change import requirements, that could be thought of as a change in api, and I'm willing to drop this for now.